src/Compilers/CSharp/Test/Symbol/Symbols/SymbolEqualityTests.cs

```diff
@@ -911,6 +920,179 @@ public static void M(A<T> t)
                 );
         }
 
+        [Fact]
+        public void TestImplementationAttributes_Inlining()
```

Comment:
>TestImplementationAttributes_Inlining [](start = 20, length = 37)

I do not expect the new API to have any impact on symbol equality. Any changes in this file are unexpected to me.

Reply:
@AlekseyTs Any recommendation where the tests should go?

Reply:
> Any recommendation where the tests should go?

I think a new file under src\Compilers\CSharp\Test\Symbol\Symbols\ would be fine, or a "good" existing file there. Same for VB.

---
In reply to: [569592234](https://github.com/dotnet/roslyn/pull/50560#discussion_r569592234) [](ancestors = 569592234)